src/AddIns/Uno.UI.Svg/buildTransitive/Uno.UI.Svg.targets

```diff
@@ -22,10 +22,10 @@
 		</PropertyGroup>
 		<ItemGroup>
```

Comment:
For now it's probably best to take this PR as-is, but just noting a potential future improvement.

Reply:
Automatically adding package references generally causes issues because nuget cannot unconditionally detect that a reference is needed, causing a need for double restore.